src/ValueAndReferenceTypes/Trainee.cs

```diff
@@ -0,0 +1,93 @@
+﻿// <copyright file="Trainee.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ValueAndReferenceTypes
+{
+    /// <summary>
+    /// Trainee class - Gets id, name and tech stack of the intern
+    /// </summary>
+    public class Trainee
+    {
+        private enum OperationToPerform
+        {
+            Add = 1,
+            Modify = 2,
+            Display = 3,
+        }
+
+        /// <summary>
+        /// Gets or Sets Id
+        /// </summary>
+        /// <value> Id - id of the employee</value>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Gets or Sets Name
+        /// </summary>
+        /// <value>Name - name of the employee</value>
+        public string Name { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or Sets TechStack
+        /// </summary>
+        /// <value>TechStack - job title of the employee</value>
+        public string TechStack { get; set; } = string.Empty;
+
+        /// <summary>
+        /// AddDetails() function - Gets input and sets value
+        /// </summary>
+        public void AddDetails()
+        {
+            Console.WriteLine("Enter Id of the trainee: ");
+            string? unparsedTraineeId = Console.ReadLine();
+            this.Id = Program.GetValidInput(unparsedTraineeId);
+            Console.WriteLine("Enter name of the trainee: ");
+            string? traineeName = Console.ReadLine();
+            this.Name = Program.GetValidString(traineeName);
+            Console.WriteLine("Enter tech stack of the trainee: ");
+            string? traineeTechStack = Console.ReadLine();
+            this.TechStack = Program.GetValidString(traineeTechStack);
+        }
+
+        /// <summary>
+        /// PerformTraineeOperations() - Creates trainee object and performs user specific functions
+        /// </summary>
+        public void PerformTraineeOperations()
+        {
+            int userChoice = Program.GetChoiceOfOperation();
+            switch (userChoice)
+            {
+                case (int)OperationToPerform.Add:
+                    this.AddDetails();
+                    break;
+                case (int)OperationToPerform.Modify:
+                    this.ModifyDetails();
+                    break;
+                case (int)OperationToPerform.Display:
+                    this.DisplayDetails();
+                    break;
+            }
```

Comment:
Its Good to have default case
```suggestion
       default:
            break;
            }
```